modules/consensus.go

```diff
@@ -136,6 +136,11 @@ type ConsensusSet interface {
 	// run any required closing routines.
 	Close() error
 
+	// ConsensusChange returns the consensus change that occured at the input
+	// index. An error is returned if there is not a change at the provided
+	// index.
+	ConsensusChange(int) (ConsensusChange, error)
```

Comment:
What does "index" mean in this context? How is the caller supposed to know what index to use?